nginx.conf

```diff
@@ -9,4 +8,7 @@ server {
         try_files $uri/index.html $uri.html $uri/ $uri =404;
     }
 
+    location = /docs {
+        return 301 /docs/ ;
```

Comment:
This redirect ditches the URL's port component, so I get sent from `localhost:4000/docs` to `localhost/docs` where there is no server running.

Separately, if I go directly to `http://localhost:4000/docs/` it seems there's nothing there? I get a 404.

Reply:
Thanks for catching this. I talked to Starchy about it today - he will add this rule to the load balancer rather than the nginx container. Which also means at this point this nginx.conf is really just a convenience for building the site with Docker and not a a mirror of the production container.

For the development version, I went ahead an made the ports match inside the container and outside the container again.